code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -697,6 +697,7 @@
 /mob/living/simple_animal/parrot/proc/parrot_hear(message)
 	if(!message || stat)
 		return
+	message = strip_html_tags(message)
 	speech_buffer.Add(message)
```

Comment:
Speed 😎 
```suggestion
	speech_buffer.Add(strip_html_tags(message))
```

Reply:
that probably a way better way to do it, yeah